exercises/accumulate/README.md

```diff
@@ -33,6 +33,10 @@ This exercise requires you to write an extension method. For more information, s
 
 To run the tests, run the command `dotnet test` from within the exercise directory.
 
+### Laziness test
+
+Since `accumulate` returns an IEnumerable it's execution can be deferred until `ToList()` it is called on it and this is tested with the `Accumulate_is_lazy` method
```

Comment:
Some tiny nits:
- Maybe enclose `IEnumerable` in backticks? (to have it render as code)
- I think I would prefer a comma after `IEnumerable`.
- `can be deferred` -> `is deferred`
- `called on it and this is` -> `called on it, which is`

Reply:
sure, let me fix those

Reply:
Great!